--- resource-agents-4.1.1/tools/sfex_lib.c.orig	2018-03-05 10:25:31.000000000 +0100
+++ resource-agents-4.1.1/tools/sfex_lib.c	2018-09-02 20:08:37.594025340 +0200
@@ -426,7 +426,7 @@
     return -1;
   }
   ldata->count = atoi ((char *) (block->count));
-  strncpy ((char *) (ldata->nodename), (const char *) (block->nodename), sizeof(block->nodename));
+  strncpy ((char *) (ldata->nodename), (const char *) (block->nodename), sizeof(ldata->nodename));
 
 #ifdef SFEX_DEBUG
   cl_log(LOG_INFO, "status: %c\n", ldata->status);
