--- ClusterLabs-resource-agents-b735277/configure.ac~	2011-06-29 14:51:48.000000000 +0200
+++ ClusterLabs-resource-agents-b735277/configure.ac	2012-07-04 18:29:56.185645216 +0200
@@ -366,12 +366,6 @@
 
 AC_CHECK_HEADERS(heartbeat/glue_config.h)
 
-if test "$ac_cv_header_heartbeat_glue_config_h" = "yes"; then
-  OCF_ROOT_DIR=`extract_header_define heartbeat/glue_config.h OCF_ROOT_DIR`
-else
-  enable_libnet=no
-fi
-
 AC_DEFINE_UNQUOTED(OCF_ROOT_DIR,"$OCF_ROOT_DIR", OCF root directory - specified by the OCF standard)
 AC_SUBST(OCF_ROOT_DIR)
 
