--- resource-agents-4.5.0/configure.ac.orig	2020-05-01 15:46:08.451797734 +0200
+++ resource-agents-4.5.0/configure.ac	2020-05-01 15:51:52.173268968 +0200
@@ -23,8 +23,6 @@
 	m4_esyscmd([make/git-version-gen .tarball-version]),
 	[developers@clusterlabs.org])
 
-AC_USE_SYSTEM_EXTENSIONS
-
 CRM_DTD_VERSION="1.0"
 
 PKG_FEATURES=""
@@ -33,6 +31,7 @@
 AC_CONFIG_MACRO_DIR([m4])
 
 AC_CANONICAL_HOST
+AC_USE_SYSTEM_EXTENSIONS
 
 dnl Where #defines go (e.g. `AC_CHECK_HEADERS' below)
 dnl
