diff -dur resource-agents-3.9.7.orig/heartbeat/apache resource-agents-3.9.7/heartbeat/apache
--- resource-agents-3.9.7.orig/heartbeat/apache	2016-02-03 11:57:55.000000000 +0100
+++ resource-agents-3.9.7/heartbeat/apache	2016-11-28 10:10:14.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 #	High-Availability Apache/IBMhttp control script
 # 
diff -dur resource-agents-3.9.7.orig/heartbeat/CTDB resource-agents-3.9.7/heartbeat/CTDB
--- resource-agents-3.9.7.orig/heartbeat/CTDB	2016-02-03 11:57:55.000000000 +0100
+++ resource-agents-3.9.7/heartbeat/CTDB	2016-11-28 10:10:14.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 #             OCF Resource Agent for managing CTDB
 #
diff -dur resource-agents-3.9.7.orig/heartbeat/dhcpd resource-agents-3.9.7/heartbeat/dhcpd
--- resource-agents-3.9.7.orig/heartbeat/dhcpd	2016-02-03 11:57:55.000000000 +0100
+++ resource-agents-3.9.7/heartbeat/dhcpd	2016-11-28 10:10:14.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 #   Resource Agent for managing dhcpd resources.
 #
diff -dur resource-agents-3.9.7.orig/heartbeat/exportfs resource-agents-3.9.7/heartbeat/exportfs
--- resource-agents-3.9.7.orig/heartbeat/exportfs	2016-02-03 11:57:55.000000000 +0100
+++ resource-agents-3.9.7/heartbeat/exportfs	2016-11-28 10:10:14.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # exportfs
 #
 # Description: Manages nfs exported file system.
diff -dur resource-agents-3.9.7.orig/heartbeat/nginx resource-agents-3.9.7/heartbeat/nginx
--- resource-agents-3.9.7.orig/heartbeat/nginx	2016-02-03 11:57:55.000000000 +0100
+++ resource-agents-3.9.7/heartbeat/nginx	2016-11-28 10:10:14.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 #	High-Availability nginx OCF resource agent
 # 
diff -dur resource-agents-3.9.7.orig/heartbeat/pgsql resource-agents-3.9.7/heartbeat/pgsql
--- resource-agents-3.9.7.orig/heartbeat/pgsql	2016-02-03 11:57:55.000000000 +0100
+++ resource-agents-3.9.7/heartbeat/pgsql	2016-11-28 10:09:54.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Description:  Manages a PostgreSQL Server as an OCF High-Availability
 #               resource
diff -dur resource-agents-3.9.7.orig/heartbeat/tomcat resource-agents-3.9.7/heartbeat/tomcat
--- resource-agents-3.9.7.orig/heartbeat/tomcat	2016-02-03 11:57:55.000000000 +0100
+++ resource-agents-3.9.7/heartbeat/tomcat	2016-11-28 10:10:14.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # Description:  Manages a Tomcat Server as an OCF High-Availability
 #               resource under Heartbeat/LinuxHA control
diff -dur resource-agents-3.9.7.orig/heartbeat/Xen resource-agents-3.9.7/heartbeat/Xen
--- resource-agents-3.9.7.orig/heartbeat/Xen	2016-02-03 11:57:55.000000000 +0100
+++ resource-agents-3.9.7/heartbeat/Xen	2016-11-28 10:10:14.000000000 +0100
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 #
 # Support:      linux-ha@lists.linux-ha.org
