diff -ur resource-agents-4.14.0.orig/tools/storage_mon.c resource-agents-4.14.0/tools/storage_mon.c
--- resource-agents-4.14.0.orig/tools/storage_mon.c	2024-04-24 09:11:40.000000000 +0000
+++ resource-agents-4.14.0/tools/storage_mon.c	2024-04-25 19:29:01.248095163 +0000
@@ -1,3 +1,4 @@
+#include <inttypes.h>
 #include <stdio.h>
 #include <getopt.h>
 #include <stdlib.h>
@@ -150,7 +151,7 @@
 		goto error;
 	}
 	if (verbose) {
-		PRINT_STORAGE_MON_INFO("%s: opened %s O_DIRECT, size=%zu", device, (flags & O_DIRECT)?"with":"without", devsize);
+		PRINT_STORAGE_MON_INFO("%s: opened %s O_DIRECT, size=%"PRIu64, device, (flags & O_DIRECT)?"with":"without", devsize);
 	}
 
 	/* Don't fret about real randomness */
@@ -163,7 +164,7 @@
 		goto error;
 	}
 	if (verbose) {
-		PRINT_STORAGE_MON_INFO("%s: reading from pos %ld", device, seek_spot);
+		PRINT_STORAGE_MON_INFO("%s: reading from pos %"PRId64, device, (int64_t)seek_spot);
 	}
 
 	if (flags & O_DIRECT) {
@@ -203,7 +204,7 @@
 			goto error;
 		}
 		if (res < (int)sizeof(buffer)) {
-			PRINT_STORAGE_MON_ERR("Failed to read %ld bytes from %s, got %d", sizeof(buffer), device, res);
+			PRINT_STORAGE_MON_ERR("Failed to read %zd bytes from %s, got %d", sizeof(buffer), device, res);
 			goto error;
 		}
 	}
