--- resource-agents-4.10.0/configure.ac.orig	2021-11-03 10:12:01.000000000 +0100
+++ resource-agents-4.10.0/configure.ac	2022-02-07 20:11:41.207130114 +0100
@@ -23,14 +23,13 @@ AC_INIT([resource-agents],
 	m4_esyscmd([make/git-version-gen .tarball-version]),
 	[developers@clusterlabs.org])
 
-AC_USE_SYSTEM_EXTENSIONS
-
 CRM_DTD_VERSION="1.0"
 
 AC_CONFIG_AUX_DIR(.)
 AC_CONFIG_MACRO_DIR([m4])
 
 AC_CANONICAL_HOST
+AC_USE_SYSTEM_EXTENSIONS
 
 dnl Where #defines go (e.g. `AC_CHECK_HEADERS' below)
 dnl
