--- resource-agents-4.10.0/tools/storage_mon.c.orig	2021-11-03 10:12:01.000000000 +0100
+++ resource-agents-4.10.0/tools/storage_mon.c	2022-02-07 18:30:47.352605699 +0100
@@ -1,3 +1,4 @@
+#include <inttypes.h>
 #include <stdio.h>
 #include <getopt.h>
 #include <stdlib.h>
@@ -60,7 +61,7 @@ static void *test_device(const char *dev
 		exit(-1);
 	}
 	if (verbose) {
-		fprintf(stderr, "%s: size=%zu\n", device, devsize);
+		fprintf(stderr, "%s: size=%"PRIu64"\n", device, devsize);
 	}
 	/* Don't fret about real randomness */
 	srand(time(NULL) + getpid());
@@ -74,7 +75,7 @@ static void *test_device(const char *dev
 	}
 
 	if (verbose) {
-		printf("%s: reading from pos %ld\n", device, seek_spot);
+		printf("%s: reading from pos %"PRId64"\n", device, (int64_t)seek_spot);
 	}
 
 	res = read(device_fd, buffer, sizeof(buffer));
@@ -84,7 +85,7 @@ static void *test_device(const char *dev
 		exit(-1);
 	}
 	if (res < (int)sizeof(buffer)) {
-		fprintf(stderr, "Failed to read %ld bytes from %s, got %d\n", sizeof(buffer), device, res);
+		fprintf(stderr, "Failed to read %zd bytes from %s, got %d\n", sizeof(buffer), device, res);
 		close(device_fd);
 		exit(-1);
 	}
